docs/howto/write-scenario-tests-for-a-charm.md

```diff
@@ -1,45 +1,67 @@
 (write-scenario-tests-for-a-charm)=
-# How to write scenario tests for a charm
+# How to write unit tests for a charm
 
-First of all, install scenario:
+First of all, install the testing framework. To do this in a virtual environment
+while you are developing, you can do this with `pip` or any other package
+manager; for example:
 
-`pip install ops-scenario`
+```
+pip install ops[testing]
+```
+
+Normally, you'll include this in the dependency group for your unit tests, for
+example in a test-requirements.txt file:
+
+```text
+ops[testing] ~= 2.17
+```
+
+Or in `pyproject.toml`:
+
+```toml
+[dependency-groups]
+test = [
+  "ops[testing] ~= 2.17",
+]
+```
 
 Then, open a new `test_foo.py` file where you will put the test code.
 
 ```python
-# import the necessary objects from scenario and ops
-from scenario import State, Context
 import ops
+from ops import testing
 ```
 
-
 Then declare a new charm type:
 ```python
 class MyCharm(ops.CharmBase):
     pass        
 ```
-And finally we can write a test function. The test code should use a Context object to encapsulate the charm type being tested (`MyCharm`) and any necessary metadata, then declare the initial `State` the charm will be presented when run, and `run` the context with an `event` and that initial state as parameters. 
+
+And finally we can write a test function. The test code should use a `Context` object to encapsulate the charm type being tested (`MyCharm`) and any necessary metadata, then declare the initial `State` the charm will be presented when run, and `run` the context with an `event` and that initial state as parameters. 
+
 In code:
 
 ```python
 def test_charm_runs():
-    # arrange: 
-    #  create a Context to specify what code we will be running
-    ctx = Context(MyCharm, meta={'name': 'my-charm'})
-    #  and create a State to specify what simulated data the charm being run will access
-    state_in = State()
-    # act:
-    #  ask the context to run an event, e.g. 'start', with the state we have previously created
+    # Arrange: 
+    #  Create a Context to specify what code we will be running,
+    ctx = testing.Context(MyCharm)
+    #  and create a State to specify what simulated data the charm being run will access.
+    state_in = testing.State()
```

Comment:
Might be good to have at least one parameter to `State()` for clarity, even just `leader=True`.